extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/ServerSideEncryptingAmazonS3.java

```diff
@@ -58,11 +61,19 @@ public static Builder builder()
 
   private final AmazonS3 amazonS3;
   private final ServerSideEncryption serverSideEncryption;
+  private TransferManager transferManager = null;
```

Comment:
this should be made final and initialized in the constructor

Reply:
code updated